migrations/V9__Plan_Collaborator_Lead_Trigger.sql

```diff
@@ -0,0 +1,29 @@
+CREATE FUNCTION collaborator_role_check_trigger() RETURNS TRIGGER AS $role_check$
+BEGIN
+ IF (SELECT count(*)
+ FROM plan_collaborator
+ WHERE  team_role = 'MODEL_LEAD' and model_plan_id = OLD.model_plan_id 
+ )<2
```

Comment:
Why is this `<2` instead of `<1`?

Reply:
less than 2 captures that there is only 1 `MODEL_LEAD` selected. <1 means there aren't any `MODEL_LEAD`s. 

Otherwise we could represent this as <= 1. 

This is important because this function is used as a `BEFORE` trigger, so the check is happening before we make a change instead, of checking after a change and rolling back.

Reply:
Ahh - the face that it's a `BEFORE` helps a ton. Would you mind just chucking a quick comment on this for anyone looking in the future?